plugin.cfg

```diff
@@ -20,6 +20,7 @@
 # log:log
 
 metadata:metadata
+geoip:geoip
```

Comment:
I didn't know what the right position would be, I had the impression for this plugin (geoip) it doesn't matter much since its serveDNS function just passes the chain?

Reply:
this looks fine